docs/notebooks/hydrological_modelling.ipynb

```diff
@@ -98,8 +98,8 @@
     "- `.get_inputs()` to retrieve the meteorological inputs.\n",
```

Comment:
<p>Once #212 is merged, we can replace this code with:</p>

<pre class="ql-syntax" spellcheck="false">from xhydro.testing.helpers import deveraux
import xarray as xr

meteo_file = "hydro_modelling/ERA5_testdata.nc"
ds = xr.open_dataset(deveraux.fetch(meteo_file))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Leave this for now. The other branch will add this change in!